cloudbuild.yaml

```diff
@@ -202,4 +163,3 @@ steps:
       stack exec -- radicle-daemon &
 
       stack test :e2e
-      stack exec -- radicle test/machine-backends.rad radicle-server
```

Comment:
Please also remove the corresponding file `test/machine-backends.rad`. Otherwise we’ll forget about it

Reply:
:+1: